website/orders/services.py

```diff
@@ -97,8 +99,8 @@ def add_order(product, shift, order_type, user=None, set_done=False, force=False
             type=order_type,
             user=user,
             user_association=user.profile.association,
-            paid=set_done,
-            ready=set_done,
+            paid=paid,
```

Comment:
Again, do we need to set 'paid' and 'ready'?

Reply:
I reverted it to setting paid and ready later.